src/components/Navigation.tsx

```diff
@@ -38,14 +38,18 @@ function NavLink({
   active?: boolean
   isAnchorLink?: boolean
 }) {
+  const isActive = usePathname().startsWith(href); //checking if the pathname starts with the link's href.
```

Comment:
We should remove `isActive` or `active`. It is confusing to have both `isActive` and `active` variables when only one is being used.

Reply:
Remove active variable